### ISSUE TITLE
Failed to load into SQLite

### ISSUE THREAD

**Issue by CodeInFilth** [spartan737/Stocksera#10]
sqlite3.IntegrityError: UNIQUE constraint failed: subreddit_count.ticker, subreddit_count.subreddit, subreddit_count.updated_date

Would this be fixed by clearing the database?

**Comment by spartan737** [comment]
It seems like you have run it 2 times. and there are no new redditors. No worries. Replace line 132 with

`db.execute("INSERT OR IGNORE INTO subreddit_count VALUES (?, ?, ?, ?, ?, ?, ?)", `

instead (add the OR IGNORE).

**Comment by CodeInFilth** [comment]
Thanks adjusting some settings and running into this

`sqlite3.OperationalError: table short_volume has 6 columns but 7 values were supplied`

**Comment by spartan737** [comment]
I created a new table for short_volume. Delete the table in the database and rerun scheduled_tasks/create_database.py. In the future, please use the latest commit to reduce error.

**Comment by CodeInFilth** [comment]
Great ! Everything you have directed me towards was a fix!

After getting an error when I finally merged the latest commit and any changes not included I got this error. 
```
Traceback (most recent call last):
  File "tasks_to_run.py", line 80, in <module>
    get_subreddit_count.subreddit_count()
  File "/root/_startUp/Stocksera/scheduled_tasks/reddit/get_subreddit_count.py", line 133, in subreddit_count
    (date_updated, key, subreddit_name, subscribers, active, percentage_active, growth))
sqlite3.IntegrityError: UNIQUE constraint failed: subreddit_count.ticker, subreddit_count.subreddit, subreddit_count.updated_date
```

Luckily this is an easy one off the top of my head so I went through all of the scripts in scheduled_tasks only to find  either `INSERT OR REPLACE` or `INSERT` alone.

After change all the `INSERT` into `INSERT OR REPLACE`


The only other issue I run into quiet often is scraping the Crypto information from coingecko and timing out, so to fix this I have been adding a time.sleep(2) (a 2second sleep seems to be enough for the API) on line _297_ above your #comment for saving graphs on 35 symbols only.

Hope these small notes help as other bugs were easy and seemed to be user specific considering I am testiner on either Ubuntu 18.04 or Mac OS 10.14 and your using Windows I believe so its pretty straight forward.
